crates/nargo_cli/tests/test_data_ssa_refactor/brillig_oracle/src/main.nr

```diff
@@ -1,10 +1,16 @@
+use dep::std::slice;
+
 // Tests oracle usage in brillig/unconstrained functions
 fn main(x: Field)  {
     // call through a brillig wrapper
     oracle_print_array_wrapper([x, x]);
 
     // TODO(#1615) Nargo currently only supports resolving one foreign call
     // oracle_print_wrapper(x);
+
+    // let slice = get_number_sequence_wrapper(12);
+    // assert(slice.len() == 12);
+    get_number_sequence_wrapper(20);
```

Comment:
Is the commented code here not working?

Reply:
No we need more logic in ACIR gen to enable returning slices from a Brillig function itself. What @guipublic is doing right now should work towards enabling this